MonoGame.Framework/Content/ContentReaders/MultiArrayReader.cs

```diff
@@ -0,0 +1,108 @@
+﻿#region License
```

Comment:
Why has different license text been used here.  Obviously it's jus comments but makes the repo look messy.  Should use the same for all files.